README.md

```diff
@@ -1,48 +1,44 @@
-[![npm version](https://img.shields.io/npm/v/@gnosis.pm/gp-v1-ui.svg?style=flat)](https://npmjs.org/package/@gnosis.pm/gp-v1-ui 'View this project on npm')
-&nbsp;
-[![Build Status](https://travis-ci.org/gnosis/gp-v1-ui.svg?branch=develop)](https://travis-ci.org/gnosis/gp-v1-ui)
-&nbsp;
-[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1-ui/badge.svg?branch=master)](https://coveralls.io/github/gnosis/gp-v1-ui?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1/badge.svg?branch=master)](https://coveralls.io/github/gnosis/gp-v1?branch=master)
 
 Develop:
-&nbsp;
-[![Build Status](https://travis-ci.org/gnosis/gp-v1-ui.svg?branch=develop)](https://travis-ci.org/gnosis/gp-v1-ui)
-&nbsp;
-[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1-ui/badge.svg?branch=develop)](https://coveralls.io/github/gnosis/gp-v1-ui?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1/badge.svg?branch=develop)](https://coveralls.io/github/gnosis/gp-v1?branch=develop)
 
-# Gnosis Protocol v1 Web
+# Gnosis Protocol UI
 
-> IMPORTANT: This repository is for the legacy Gnosis Protocol v1
-> For the protocol latest version UI go to: https://github.com/gnosis/gp-ui
-
-<img align="right" width="350" src="./docs/screenshot.png">
-
-`Gnosis Protocol v1 Web` is the first dapp built on the [Gnosis Protocol](https://docs.gnosis.io/protocol).
+`Gnosis Protocol UI` contains some reusable components for [Gnosis Protocol](https://docs.gnosis.io/protocol) (`🚨 WARNING: the doc might refer still to v1 of the protocol`).
 
 Gnosis Protocol is a fully permissionless DEX that enables ring trades to maximize liquidity.
 
-`Gnosis Protocol v1 Web` allows users to:
-
-- Create orders in [Gnosis Protocol](https://docs.gnosis.io/protocol)
-- See the balances for any token: available in their wallet and on the exchange
-- Deposit and withdraw tokens to/from the exchange wallet
+`Gnosis Protocol UI` contains:
 
-## 📚 Wiki
+- **Explorer**: Gnosis Protocol explorer. Allows to explore the protocol orders, trades. For now is a WIP 👷‍♀️.
+- **Trade**: Classical trading interface. For now is just a WIP 👷‍♀️. 
+- **Legacy GP v1 UI**: Trading interface for Gnosis Protocol v1. It's the only app that is not generated automatically, but it can be run or build by following the instructions in this readme.
```

Comment:
```suggestion
- **Legacy GP v1 UI**: Trading interface for Gnosis Protocol v1. It's the only app that is not generated automatically, but it can be run/built by following the instructions in this readme.
```